lib/cpp/build/install-libcxx.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env sh
+set -e
+
+git clone --depth 1 --branch llvmorg-12.0.1 https://github.com/llvm/llvm-project.git
+
+cd llvm-project
+mkdir build
+CODEGEN=1 NO_LLVM=1 CC=~/.symbolica/bin/cc CXX=~/.symbolica/bin/clang++ \
```

Comment:
I think we might want `CXXFLAGS="-fno-threadsafe-statics"` here so that the bitcode doesn't generate `__cxa_guard_acquire` and friends.

Reply:
Well let's test it first and see if we actually do hit it in any user code.